packages/datastore/package.json

```diff
@@ -17,16 +17,12 @@
     "@phosphor/coreutils": "^1.3.0",
     "@phosphor/disposable": "^1.1.2",
     "@phosphor/messaging": "^1.2.2",
-    "@phosphor/signaling": "^1.2.2",
-    "node-fetch": "^2.1.2",
-    "ws": "^5.2.0"
+    "@phosphor/signaling": "^1.2.2"
   },
   "devDependencies": {
-    "@types/node-fetch": "^2.1.0",
-    "@types/ws": "^5.1.1",
     "rimraf": "^2.5.2",
     "typedoc": "^0.7.0",
-    "typescript": "~2.9.0"
+    "typescript": "~3.3.1"
```

Comment:
Can you use the same version of typescript that's currently being used in master? Then we can update all packages at once after we merge this branch into master.